staging/src/k8s.io/client-go/tools/clientcmd/api/v1/conversion.go

```diff
@@ -172,3 +172,9 @@ func Convert_Map_string_To_runtime_Object_To_Slice_v1_NamedExtension(in *map[str
 	}
 	return nil
 }
+
+func Convert_api_ExecConfig_To_v1_ExecConfig(in *api.ExecConfig, out *ExecConfig, s conversion.Scope) error {
+	// This intentionally omits the in.Config field which is an internal-only field for storing
```

Comment:
I was getting the following error when doing codegen, so I added this manual conversion.
```
E1026 16:32:58.380934   33931 conversion.go:750] Warning: could not find nor generate a final Conversion function for k8s.io/client-go/tools/clientcmd/api.ExecConfig -> k8s.io/client-go/tools/clientcmd/api/v1.ExecConfig
E1026 16:32:58.381097   33931 conversion.go:751]   the following fields need manual conversion:
E1026 16:32:58.381105   33931 conversion.go:753]       - Config
```